src/settings/setDelay.ts

```diff
@@ -0,0 +1,22 @@
+import { Setting } from "obsidian";
+import FileHider from "../main";
+
+export class SetDelaySetting {
+
+	public static create(plugin: FileHider, container: HTMLElement) {
+		return new Setting(container)
+		.setName(`Set Delay`)
+		.setDesc(`Set the delay for hiding after explorer loads.`)
+		.addText(text => {
+			text
+			.setValue(plugin.settings.delay.toString())
```

Comment:
The number of milliseconds isn't really a great way to expose this setting (especially with no indication that it's milliseconds); I would prefer to see it presented as `YsZms` ("Y seconds, Z milliseconds", where each part of it is optional).

I think going to seconds is as much as is needed, at _most_, I'd allow minutes. Of course with this change it would also require updating the description to be more descriptive.

---

The setting should still store the value as milliseconds in the config though :)